### PR TITLE
Learn products component structure

### PR DESCRIPTION
**Pull request by chasulin**

### CODE REVIEW

File: src/screens/Learn/Products/ProductPage.tsx [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+    return null;
+  }
+
+  const {
+    productName,
+    productIntro,
+    metadataTitle,
+    metadataDescription,
+    sections,
+  } = pageContent[0];
```
Comment (pnavarrc) [inline src/screens/Learn/Products/ProductPage.tsx:30]: We probably need a map to fetch pageContent given a product ID (like we did for the case studies). We can do this on a follow-up PR to unblock content entry

**Review by pnavarrc — APPROVED** [review]
Just a small comment!

File: src/screens/Learn/Products/ProductPage.tsx (hunk 1 above)
Comment (chasulin) [reply src/screens/Learn/Products/ProductPage.tsx:30]: that is already set up. `pageContent` is the return value of `getProductPageContent(productId)` a few lines up, which is the function I put in `learn/index` (image below). i used `.filter` so i just needed to grab the obj from the return array, hence the `[0]`

Screen Shot 2020-11-09 at 7 36 18 PM

Comment (pnavarrc) [reply src/screens/Learn/Products/ProductPage.tsx:30]: oh, how about using `find` instead then?

**Pull request merged**